src/api/app/views/webui2/shared/bs_requests/index.json.erb

```diff
@@ -0,0 +1,23 @@
+{
+  "draw": "<%= @requests_data_table.draw %>",
+  "recordsTotal": "<%= @requests_data_table.records_total %>",
+  "recordsFiltered": "<%= @requests_data_table.count_requests %>",
+  "data": [
+    <% @requests_data_table.rows.each.with_index do |row,index | %>
+      [
+        "<%= escape_javascript(fuzzy_time(row.created_at, false)) %>",
+        <% cache "request-table-row-#{row.id}-#{row.updated_at.to_i}" do %>
+          "<%= escape_javascript(project_or_package_link(project: row.source_project, package: row.source_package, creator: row.creator, trim_to: 40, short: true)) %>",
+          "<%= escape_javascript(target_project_link(row)) %>",
+          "<%= escape_javascript(user_with_realname_and_icon(row.creator, short: true)) %>",
+          "<%= escape_javascript(new_or_update_request(row)) %>",
+          "<%= escape_javascript(row.priority) %>",
+          "<%= escape_javascript(link_to(icon('fa', 'info-circle'), request_show_path(row.number), { title: "Show request ##{row.number}", class: :request_link })) %>"
```

Comment:
Oh please avoid the bloody helpers if you can. I would really appreciate it if we could get rid of all this nonsense and pull only over what we really need...

But we can do this later, I'm fine with a FIXME...

Reply:
The `icon` call? What's so bad about it?

Reply:
I see, though [icon](https://github.com/FortAwesome/font-awesome-sass#rails-helper-usage) is a at least not a custom one.